### PR TITLE
[plugin.video.mediathekview] 0.6.2

### PR DESCRIPTION
**Pull request by tuxpoldo**
### Description
- Implemented Workaround for broken `gzip` module support in Kodi 18 running on Android
- Fixed crash in `mvupdate` tool when invoked with MySQL support
- Improved compatibility with future Kodi versions with Python 3 support
- Fixed module exception due to case error
### Checklist:

- [X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [X] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

File: plugin.video.mediathekview/addon.xml [hunk 1]
```diff
@@ -26,7 +26,11 @@
 		<description lang="de_DE">Ermöglicht den Zugriff auf fast alle deutschen Mediatheken der öffentlich Rechtlichen basierend auf der Datenbank von MediathekView.de</description>
 		<description lang="en_GB">Gives access to most video-platforms from German public service broadcasters using the database of MediathekView.de</description>
 		<description lang="it_IT">Fornisce l'accesso a gran parte delle piattaforme video operate dalle emittenti pubbliche tedesche usando la banca dati di MediathekView.de</description>
-		<news>v.0.6.1 (2019-03-08):
+		<news>v.0.6.2 (2019-03-10):
+- Bugfix: Entpacken von GZ-Archiven auf FireTV 4K und ähnliche Geräte funktioniert nun wieder
```
Comment (enen92) [inline plugin.video.mediathekview/addon.xml]: English please...this is shown in the interface

**Comment by tuxpoldo** [comment]
Done!

File: plugin.video.mediathekview/addon.xml (hunk 1 above)
Comment (tuxpoldo) [reply plugin.video.mediathekview/addon.xml]: Done.

**Comment by Rechi** [comment]
@tuxpoldo this add-on isn't Python 3 compatible. Future updates have to be compatible with Python 3.
Have a look at https://kodi.tv/article/attention-addon-developers-migration-python-3 for more information.

**Pull request merged**